src/api/makeup-artiste/content-types/makeup-artiste/schema.json

```diff
@@ -105,6 +105,9 @@
     "pro": {
       "type": "boolean",
       "default": false
+    },
+    "company_artist_name": {
+      "type": "string"
```

Comment:
Tu peux ajouter une valeur pas défaut a "" ( en mode string vide ) ?

Reply:
bien vu

Reply:
on en a pas du tout en vrai sur les string

Reply:
![image](https://github.com/For-Hives/api-my-makeup/assets/55706543/a15e408d-2958-48d6-aa79-828dde8b4f2e)